backend/keystore/keystore.go

```diff
@@ -39,9 +39,16 @@ type Keystore interface {
 	// This is typically done through a screen on the device or through a paired mobile phone.
 	HasSecureOutput(*signing.Configuration, coin.Coin) (bool, error)
 
-	// OutputAddress outputs the public key at the given configuration for the given coin.
+	// VerifyOutputAddress outputs the public key at the given configuration for the given coin.
 	// Please note that this is only supported if the keystore has a secure output channel.
-	OutputAddress(*signing.Configuration, coin.Coin) error
+	VerifyOutputAddress(*signing.Configuration, coin.Coin) error
+
+	// HasSecureBTCPubOutput returns whether the keystore supports to output an xpub/zpub/tbup/ypub securely.
+	// Only supported on v2
```

Comment:
remove this, as an interface, it shouldn't know about particular instances, and this can be outdated quickly.